codes.loers.Karlender.yml

```diff
@@ -0,0 +1,36 @@
+app-id: codes.loers.Karlender
+runtime: org.gnome.Platform
+runtime-version: "master"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: codes.loers.Karlender
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-data
+  - --filesystem=/etc/localtime
+  - --filesystem=/usr/share/zoneinfo
+  - --metadata=X-DConf=migrate-path=/dev/loers/Karlender/
+  - --socket=system-bus
+  - --socket=session-bus
```

Comment:
I am struggeling with removing these. I leave the app running when a window is closed and listen to my own bus so when a new instance is started it checks via `codes.loers.karlender.AppMutex` if the app is running already and it will open its window then. I thought adding `talk-name=codes.loers.karlender.*` would suffice but it seems it does not. On starting the first and when I try to start a second instance the first (running) one throws the following error:
```
DBus failed: org.freedesktop.DBus.Error.AccessDenied
```
I also added `talk-name=org.freedesktop.*` since I need to talk to secret-service.
Do you have an advice on how to handle that?

Reply:
This mean you app-id should be `codes.loers.karlender`. It's case sensitive and by default grant this as "own-name" for dbus.